specs/Workers.md

```diff
@@ -0,0 +1,1402 @@
+Additionally, it provides a 'destroy' event that is raised just before this
+object is due for destruction.
+
+**GetSharedWorkers**: The GetSharedWorkers method, part of the
+CoreWebViewSharedWorkerManager class, is used to retrieve all shared workers
+created within the context of a WebView2 application for the profile. This
+method gives a collection of CoreWebView2SharedWorkers objects.
+
+# Background
+This API specification provides a basic overview, including fundamental concepts
+and introduces management APIs for interaction.
+Please note that while this specification does not add new APIs, future API
+specifications will introduce additional features for workers.
+
+Stay tuned for updates to this spec and look out for new API specs that will
+expand on the capabilities of workers. 
+
+See work in progress Workers API specs
+ - [API Spec for SW Background Sync](https://github.com/MicrosoftEdge/WebView2Feedback/blob/api-periodic-sync-and-background-sync-draft/specs/PeriodicBackgoundSyncAndBackgroundSync.md)
+
+# Examples
+## Dedicated Worker
+## Monitoring the Creation/Destruction of Dedicated Workers.
+The following example demonstrates how to subscribe to the event that is raised
+when a web page creates a dedicated worker. This event provides a
+`CoreWebView2DedicatedWorker` object, enabling the host application to interact
+with it.
+
+This example also showcases the utilization of the upcoming PostMessage and
+WorkerMessageReceived APIs. These APIs enable efficient communication between
+the main thread and a dedicated worker.
+
+## .NET/WinRT
+```c#
+    void PostMessageToWorker(CoreWebView2DedicatedWorker worker)
+    {
+        // Inside the worker, an event listener is set up for the 'message' event.
+        // When a message is received from the host application, the worker
+        // performs a fetch request. The host application also sets up an event
+        // listener to receive messages from the worker. These messages could
+        // contain fetched data or any error information from the worker's fetch
+        // operation.
+        worker.WorkerMessageReceived += (sender, args) =>
+        {
+            var message = args.TryGetWorkerMessageAsString;
+            // This section of the code is dedicated to handling updates received
+            // from the worker. Depending on the nature of these updates, you
+            // might choose to perform various actions. For instance, you could
+            // modify the user interface to reflect the new data, or you could
+            // log the received data for debugging or record-keeping purposes.
+        };
+
+        // You can dispatch a message to the worker with the URL you want to
+        // fetch data from the host app.
+        worker.PostMessage("{ MyMessage: { type: 'FETCH_URL',url: 'https://example.com/data.json' }");
+    }
+
+    void DedicatedWorkerCreatedExecuted(object target, ExecutedRoutedEventArgs e)
```

Comment:
This sample looks like it's from a WPF test app. Copying from working code is great, but not knowing that context it's a confusing distraction. (And knowing the context would be info overload.) This could probably be updated to remove the test-isms

Reply:
See above.